src/OrchardCore.Modules/OrchardCore.Recipes/Controllers/AdminController.cs

```diff
@@ -110,13 +121,7 @@ public async Task<ActionResult> Execute(string basePath, string fileName)
 
             try
             {
-                await _recipeExecutor.ExecuteAsync(executionId, recipe, new Dictionary<string, object>
-                {
-                    { "SiteName", site.SiteName },
-                    { "AdminUsername", User.Identity.Name },
-                    { "AdminUserId", User.FindFirstValue(ClaimTypes.NameIdentifier) }
```

Comment:
So now the admin user name / id will be populated by the related provider from the site settings super user, not from the current user that did the action, so it may be different but maybe this is before that is was wrong.

Reply:
> before that is was wrong.

I think probably it was wrong before. The `AdminUserId` should refer to the `AdminUserId`.

If someone was wanting to use `CurrentUserId` we should (or they should) implement a `CurrentUserId` property, and then use that in their recipes?

Often it will be the same user, but it just didn't seem right to me as it was.

Reply:
Okay cool then